src/main/java/org/broadinstitute/hellbender/tools/sv/PrintSVEvidence.java

```diff
@@ -135,7 +139,23 @@ public GATKPath getDrivingFeaturePath() {
     @Override
     public void onTraversalStart() {
         super.onTraversalStart();
-        initializeOutput();
+        if ( outputFilePath == null ) {
+            dumpHeader(getHeader());
+        } else {
+            initializeOutput();
+        }
+    }
+
+    private void dumpHeader( final FeaturesHeader header ) {
+        System.out.println("Dictionary:");
+        for ( final SAMSequenceRecord rec : header.getDictionary().getSequences() ) {
+            System.out.println(rec.getSAMString());
+        }
+        System.out.println();
+        System.out.println("Samples:");
+        for ( final String sampleName : header.getSampleNames() ) {
+            System.out.println(sampleName);
+        }
     }
```

Comment:
I should have been more specific about this in my review. Can you make `--sequence-dict-dump` and `--sample-list-dump` arguments (or the like) that specify file to write the header info to? This is just to be a little more machine-friendly.